docs/requirement-sets/requirement-sets.md

```diff
@@ -53,21 +53,21 @@ To work as expected, your Office Add-in might depend on a specific Office applic
-|Office on Android<sup>2</sup><br>(Microsoft 365 subscription)|- [Message Read](../manifest/extensionpoint.md#mobilemessagereadcommandsurface)<br>- [Appointment Organizer (Compose): Online meeting](../manifest/extensionpoint.md#mobileonlinemeetingcommandsurface)<br>- [Appointment Attendee (Read): Log appointment notes](../manifest/extensionpoint.md#mobilelogeventappointmentattendee)<br>- [Add-in Commands](common/add-in-commands-requirement-sets.md)<br>- [Launch Event](../manifest/extensionpoint.md#launchevent)|- [Mailbox 1.1](outlook/requirement-set-1.1/outlook-requirement-set-1.1.md)<br>- [Mailbox 1.2](outlook/requirement-set-1.2/outlook-requirement-set-1.2.md)<br>- [Mailbox 1.3](outlook/requirement-set-1.3/outlook-requirement-set-1.3.md)<br>- [Mailbox 1.4](outlook/requirement-set-1.4/outlook-requirement-set-1.4.md)<br>- [Mailbox 1.5](outlook/requirement-set-1.5/outlook-requirement-set-1.5.md)<sup>6</sup>|Not available|
+|Office on Android<sup>2 6</sup><br>(Microsoft 365 subscription)|- [Message Read](../manifest/extensionpoint.md#mobilemessagereadcommandsurface)<br>- [Appointment Organizer (Compose): Online meeting](../manifest/extensionpoint.md#mobileonlinemeetingcommandsurface)<br>- [Appointment Attendee (Read): Log appointment notes](../manifest/extensionpoint.md#mobilelogeventappointmentattendee)<br>- [Add-in Commands](common/add-in-commands-requirement-sets.md)<br>- [Launch Event](../manifest/extensionpoint.md#launchevent)|- [Mailbox 1.1](outlook/requirement-set-1.1/outlook-requirement-set-1.1.md)<br>- [Mailbox 1.2](outlook/requirement-set-1.2/outlook-requirement-set-1.2.md)<br>- [Mailbox 1.3](outlook/requirement-set-1.3/outlook-requirement-set-1.3.md)<br>- [Mailbox 1.4](outlook/requirement-set-1.4/outlook-requirement-set-1.4.md)<br>- [Mailbox 1.5](outlook/requirement-set-1.5/outlook-requirement-set-1.5.md)|Not available|
+|Office on iOS<sup>2 6</sup><br>(Microsoft 365 subscription)|- [Message Read](../manifest/extensionpoint.md#mobilemessagereadcommandsurface)<br>- [Appointment Organizer (Compose): online meeting](../manifest/extensionpoint.md#mobileonlinemeetingcommandsurface)<br>- [Appointment Attendee (Read): Log appointment notes](../manifest/extensionpoint.md#mobilelogeventappointmentattendee)<br>- [Add-in Commands](common/add-in-commands-requirement-sets.md)<br>- [Launch Event](../manifest/extensionpoint.md#launchevent)|- [Mailbox 1.1](outlook/requirement-set-1.1/outlook-requirement-set-1.1.md)<br>- [Mailbox 1.2](outlook/requirement-set-1.2/outlook-requirement-set-1.2.md)<br>- [Mailbox 1.3](outlook/requirement-set-1.3/outlook-requirement-set-1.3.md)<br>- [Mailbox 1.4](outlook/requirement-set-1.4/outlook-requirement-set-1.4.md)<br>- [Mailbox 1.5](outlook/requirement-set-1.5/outlook-requirement-set-1.5.md)|Not available|
```

Comment:
All the others start the extension point with a capital letter (e.g., Online meeting, Log appointment notes). Why is this one lowercase?

```suggestion
|Office on iOS<sup>2 6</sup><br>(Microsoft 365 subscription)|- [Message Read](../manifest/extensionpoint.md#mobilemessagereadcommandsurface)<br>- [Appointment Organizer (Compose): Online meeting](../manifest/extensionpoint.md#mobileonlinemeetingcommandsurface)<br>- [Appointment Attendee (Read): Log appointment notes](../manifest/extensionpoint.md#mobilelogeventappointmentattendee)<br>- [Add-in Commands](common/add-in-commands-requirement-sets.md)<br>- [Launch Event](../manifest/extensionpoint.md#launchevent)|- [Mailbox 1.1](outlook/requirement-set-1.1/outlook-requirement-set-1.1.md)<br>- [Mailbox 1.2](outlook/requirement-set-1.2/outlook-requirement-set-1.2.md)<br>- [Mailbox 1.3](outlook/requirement-set-1.3/outlook-requirement-set-1.3.md)<br>- [Mailbox 1.4](outlook/requirement-set-1.4/outlook-requirement-set-1.4.md)<br>- [Mailbox 1.5](outlook/requirement-set-1.5/outlook-requirement-set-1.5.md)|Not available|
```